lib/soap/request/headers.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Soap.Request.Headers do
+  @doc """
+  Headers generator by soap action and custom headers.
+
+  ## Examples
+
+      iex> wsdl = Soap.init_model("https://svn.apache.org/repos/asf/airavata/sandbox/xbaya-web/test/Calculator.wsdl", from: :url)
```

Comment:
```suggestion
      iex> wsdl = Soap.init_model("https://svn.apache.org/repos/asf/airavata/sandbox/xbaya-web/test/Calculator.wsdl", :url)
```
`Soap.init_model/2` accepts the second parameter atom and not the keyword list.